localisation/00_zuwc_united_localisation.csv

```diff
@@ -2880,7 +2873,33 @@ wonder_gundrak_name_string;Gundrak;;;;;;;;;;;;;x
 wonder_gundrak_desc_string;Gundrak is a massive complex with thick walls, tall guard towers, and stalwart gates. Inside, its structures are tiered and laden in steps winding about one another like a nest of small animals, each fitting into the next.;;;;;;;;;;;;;x
 wonder_atalhakkar_name_string;Atal'Hakkar;;;;;;;;;;;;;x
 wonder_atalhakkar_desc_string;According to legend, The Temple of Atal'Hakkar is a holy shrine dedicated to the ancient god, Hakkar the Soulflayer.;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+merge_with_champion;Merge with Champion;;;;;;;;;;;;;x
+merge_with_champion_desc;You are connected with the one who holds Frostmourne as long as a part of your soul trapped within. Though you are imprisoned in the Frozen Throne, the champion can break this ice prison and become one with you. ;;;;;;;;;;;;;x
+someone_found_frostmourne_tooltip;Someone found Frostmourne;;;;;;;;;;;;;x
+EVTDESC_WCUND_540_lich_king;[target_champion.GetBestName] couldn't reach the Frozen Throne. You no longer see [target_champion.GetHerHim]... but you feel Frostmourne is still somewhere there.;;;;;;;;;;;;;x
+EVTDESC_WCUND_540_champion;You couldn't reach the Frozen Throne. No longer you hear the voice of the Lich King, so you have no choice but to turn back.;;;;;;;;;;;;;x
+imprisoned_in_ice_tooltip;Is imprisoned within the Frozen Throne;;;;;;;;;;;;;x
+EVTDESC_WCUND_400;The [target_lich_king.GetTitle] is getting weaker, [target_lich_king.GetHerHis] power constantly seeping out of the Frozen Throne, and to prevent the Scourge from being undone an immediate action needs to be taken. As the Champion of the Scourge and wielder of Frostmourne, you must venture to Icecrown Glacier and ascend to the Frozen Throne, then claim the Helm of Domination to become a new host for the demigod and preserve control over the legions of undead lest they become a mindless horde.;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCUND_400;The [target_lich_king.GetTitle] is getting weaker, and [target_lich_king.GetHerHis] power is constantly seeping out of the Frozen Throne. In order to prevent the Scourge from being undone, immediate action needs to be taken. As the Champion of the Scourge and wielder of Frostmourne, you must venture to Icecrown Glacier and ascend to the Frozen Throne. Then, claim the Helm of Domination to become a new host for the demigod and solidify control over the legions of undead lest they become a mindless horde.;;;;;;;;;;;;;x
```